src/dots.jsx

```diff
@@ -25,8 +25,9 @@ export var Dots = React.createClass({
       slidesToScroll: this.props.slidesToScroll
     });
 
-    var dots = Array.apply(null, {length: dotCount}).map((x, i) => {
+    var dots = [];
 
+    for (var i=0; i<dotCount; i++) {
```

Comment:
Please refer to: https://github.com/akiran/react-slick/pull/109/
Thanks to @sorrycc for targeting the issue.